src/App.js

```diff
@@ -0,0 +1,32 @@
+import { Route, Routes, Link } from 'react-router-dom';
+import './App.css';
+import BooksContainer from './components/books';
+import Categories from './components/categories';
+
+function App() {
+  return (
+    <>
+      <div>
+        <nav className="navBar">
+          <h1>Bookstore CMS</h1>
+          <ul className="navUl">
+            <li className="naveLi">
+              <Link to="/">Books</Link>
+            </li>
+            <li className="naveLi">
+              <Link to="/categories">Categories</Link>
+            </li>
+          </ul>
+        </nav>
```

Comment:
- [ **OPTIONAL** ] For best practices, I think it would be better if you create a component file named `Navar.js` then add the above code lines inside. After you can import the component in the `App.js` main component.
```JavaScript
 // Exmaple

// src/components/Navbar.js file
import React from 'react';
import { Link } from 'react-router-dom';

const Navbar = () => (
   <nav className="navBar">
          <h1>Bookstore CMS</h1>
          <ul className="navUl">
            <li className="naveLi">
              <Link to="/">Books</Link>
            </li>
            <li className="naveLi">
              <Link to="/categories">Categories</Link>
            </li>
          </ul>
        </nav>
);

export default Navbar;

// src/App.js
import { Routes, Route } from 'react-router-dom';
import BooksContainer from './components/books';
import Categories from './components/categories';
import Navbar from './components/Navbar';
import './App.css';

function App() {
  return (
    <>
      <Navbar />
      <Routes>
        <Route path="/" element={<Books />} />
        <Route path="/categories" element={<Categories />} />
      </Routes>
    </>
  );
}

export default App;
```

Reply:
@tresorsawasawa  Thank you for your review, the has been resolved.